scripts/lib-franklin.js

```diff
@@ -18,8 +19,7 @@
 export function sampleRUM(checkpoint, data = {}) {
```

Comment:
Changes to `sampleRUM` are just me copying over the latest from the aem boilerplate.